README.md

```diff
@@ -11,6 +11,9 @@
 The PyPowSyBl project gives access PowSyBl Java framework to Python developers. This Python integration relies on
 GraalVM to compile Java code to a native library.
 
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=powsybl_pypowsybl-java&metric=alert_status)](https://sonarcloud.io/dashboard?id=powsybl_pypowsybl-java)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=powsybl_pypowsybl-java&metric=coverage)](https://sonarcloud.io/dashboard?id=powsybl_pypowsybl-java)
```

Comment:
I guess we can remove this if we don't check java coverage in the end.
I'm not sure where the fact that we don't check it should be specified ?